data/munin/groonga_n_records_

```diff
@@ -170,6 +170,32 @@ def setup
 end
 
 def autoconf
+  pid_path_variable_names = [
+    "pid_path",
+    "pid_file", # For backward compatibility. Remove me when 5.0.0.
+    "http_pid_path",
+    "httpd_pid_path",
+    "gqtp_pid_path",
+  ]
+  pid_paths = pid_path_variable_names.collect do |variable_name|
+    ENV[variable_name]
+  end
+  pid_paths = pid_paths.compact
+  if pid_paths.empty?
+    variable_names = pid_path_variable_names.collect do |name|
+      "env.#{name}"
+    end
+    variable_names_label = variable_names[0..-2].join(", ")
+    variable_names_label << " and/or #{variable_names.last}"
+    puts("no (No PID path is specified. Specify #{variable_names_label}.)")
+    exit(false)
+  end
+  pid_file_exist = false
+  pid_paths.each do |pid_path|
+    next unless File.exist?(pid_path)
+    pid_file_exist = true
+  end
```

Comment:
We can return `false` here when no PID file exist.

Reply:
We need similar check in `suggest` too. If `http_pid_path` doesn't exist, `suggest` doesn't print `http`. Without similar check, `suggest` will print `http` when `http://127.0.0.1/d/...` is accesible by `groonga-httpd` not `groonga --protocol http`.

Reply:
Thank you for your coments.
I've fixed them at https://github.com/groonga/groonga/pull/1363/commits/aecf24de9feabd3a0da8b854ded8ba061e82f273.